Sources/OTelSpanConventions/Generated/SpanAttributes+http.swift

```diff
@@ -0,0 +1,268 @@
+            set {
+                self.attributes = newValue.attributes
+            }
+        }
+        
+        @dynamicMemberLookup
+        public struct RequestAttributes: SpanAttributeNamespace {
+            public var attributes: SpanAttributes
+        
+            public init(attributes: SpanAttributes) {
+                self.attributes = attributes
+            }
+            /// `http.request.header`: HTTP request headers, `<key>` being the normalized HTTP Header name (lowercase), the value being the header values. 
+            /// 
+            /// - Stability: stable
+            /// 
+            /// - Type: templateStringArray
+            /// 
+            /// Instrumentations SHOULD require an explicit configuration of which headers are to be captured. Including all request headers can be a security risk - explicit configuration helps avoid leaking sensitive information. The `User-Agent` header is already captured in the `user_agent.original` attribute. Users MAY explicitly configure instrumentations to capture them even though it is not recommended. The attribute value MUST consist of either multiple header values as an array of strings or a single-item array containing a possibly comma-concatenated string, depending on the way the HTTP library provides access to headers. 
+            /// 
+            /// - Examples:
+            ///     - `http.request.header.content-type=["application/json"]`
+            ///     - `http.request.header.x-forwarded-for=["1.2.3.4", "1.2.3.5"]`
+            public var header: HeaderAttributes {
+                get {
+                    .init(attributes: self.attributes)
+                }
+                set {
+                    self.attributes = newValue.attributes
+                }
+            }
+            
+            public struct HeaderAttributes {
+                public var attributes: SpanAttributes
+            
+                public init(attributes: SpanAttributes) {
+                    self.attributes = attributes
+                }
+            
+                public mutating func set(_ key: String, to value: [String]) {
+                    let attributeId = self.attributeId(forKey: key)
+                    self.attributes[attributeId] = value
+                }
+            
+                private func attributeId(forKey key: String) -> String {
+                    var attributeId = "http.request.header."
+            
+                    for index in key.indices {
+                        let character = key[index]
+            
+                        if character == "-" {
+                            attributeId.append("_")
+                        } else {
+                            attributeId.append(character.lowercased())
+                        }
+                    }
+            
+                    return attributeId
+                }
+            }
+            public struct NestedSpanAttributes: NestedSpanAttributesProtocol {
```

Comment:
Curious that we don't get a newline here whereas one _was_ generated for the `NestedSpanAttributes` one scope above.

Reply:
The newline management is actually a bit of a headache, since we're mixing multi-line string literals, bare newline characters, and joined arrays by newlines. Given that, it might make sense to just have a final swift-format pass as the last step in the generation to keep everything consistent as mentioned here: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2#discussion_r2096990578. What do you think?

Reply:
Agreed, let's fix this using swift-format.